recipes/ctaplot/meta.yaml

```diff
@@ -0,0 +1,72 @@
+# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
+# If your package is python based, we recommend using Grayskull to generate it instead:
+# https://github.com/conda-incubator/grayskull
+
+# Jinja variables help maintain the recipe as you'll update the version only here.
+# Using the name variable with the URL in line 16 is convenient
+# when copying and pasting from another recipe, but not really needed.
+{% set name = "ctaplot" %}
+{% set version = "0.6.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 907a734af1c535148f3efde24bf56ba28ffee7e9ba7f4c171b38ed8396b81c21
+
+build:
+  noarch: python
```

Comment:
Cannot be noarch and depend on a compiler like you are doing below.  Try to use grayskull to generate this recipe, it usually creates one that is almost merge ready.

Reply:
Thanks!